app/src/main/java/com/duckduckgo/app/onboarding/ui/page/OnboardingPageFragment.kt

```diff
@@ -32,6 +34,11 @@ abstract class OnboardingPageFragment : Fragment() {
     override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? =
         inflater.inflate(layoutResource(), container, false)
 
+    override fun onAttach(context: Context) {
```

Comment:
do we need this here?

Reply:
I can move it to the `WelcomePage` fragment.